engine-src/Game/LambdaHack/Client/UI/HandleHumanGlobalM.hs

```diff
@@ -1083,11 +1094,11 @@ msgAddDone p verb = do
   b <- getsState $ getActorBody leader
   lvl <- getLevel $ blid b
   let s = case (T.words . TK.tname $ okind cotile $ lvl `at` p) of
-          [] -> "thing"
-          [x] -> x
-          (_ : xs) -> T.unwords xs
+            [] -> "thing"
+            ("open" : xs) -> T.unwords xs
```

Comment:
How about this? It's not super efficient, but does the work: if it sees that `tname` beggins with word "open", it cuts it off. Otherwise it leaves thing as they were. D'oh, now I see that in place of 
```
xs ->  T.unwords xs
```
I shoud've used `TK.tname $ okind cotile $ lvl `at` p`.

Reply:
Yes, cutting off "open" is great. And, obviously, premature optimization is evil, particularly when we are orders of magnitude faster that needed in this key-press-speed-bound part of code. Succinct code, OTOH, is invaluable.